src/DOM.js

```diff
@@ -2583,6 +2583,16 @@ class HTMLAudioElement extends HTMLMediaElement {
     }
   }
 
+  get loop() {
+    return this.audio && this.audio.loop;
```

Comment:
Could it be:

```
return this.audio ? this.audio.loop : false;
```

I don't think `undefined` is a legal value if we ever end up seeing it.

Reply:
I'll merge + patch this.